pyproject.toml

```diff
@@ -13,10 +13,19 @@ dependencies = [
 Homepage = "https://github.com/fossguild/naja"
 Repository = "https://github.com/fossguild/naja"
 
+[project.scripts]
+naja = "core:main"
+
 [dependency-groups]
 dev = [
     "pre-commit>=3.0.0",
     "black>=24.0.0",
     "ruff>=0.6.0",
     "pytest>=8.0.0",
 ]
+
+[tool.uv]
+package = true
+
+[tool.pytest.ini_options]
+pythonpath = ["src"]
```

Comment:
This fixes pythonpath when running pytest. The tests seem to be broken (for an unrelated reason), though.